test/mixins.test.js

```diff
@@ -110,6 +123,25 @@ describe('Model class', function() {
     });
   });
 
+  it('should fail to apply undefined mixin', function(done) {
+    var memory = new DataSource('mem', { connector: Memory }, modelBuilder);
+    var Item = memory.createModel('Item', { name: 'string' }, {
+      mixins: {
```

Comment:
These mixins are not needed by this test, please remove the entire `mixins:{}` section.